odl/discr/discretization.py

```diff
@@ -129,12 +129,12 @@ def __init__(self, uspace, dspace, sampling=None, interpol=None):
 
     @property
     def uspace(self):
-        """The undiscretized space."""
+        """Undiscretized space."""
```

Comment:
This could be a bit more verbose I guess, e.g. adding "of this discretization" or whatever.